src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -19,32 +19,35 @@ limitations under the License.
 
   <struct name="FabricDescriptor">
     <cluster code="0x003E"/>
-    <item name="FabricIndex" type="INT8U"/>
-    <item name="RootPublicKey" type="OCTET_STRING" length="65"/>
-    <item name="VendorId" type="INT16U"/> <!-- Change INT16U to new type VendorID #2395 -->
-    <item name="FabricId" type="FABRIC_ID"/>
-    <item name="NodeId" type="NODE_ID"/>
-    <item name="Label" type="CHAR_STRING" length="32"/>
+    <item fieldId="0" name="FabricIndex" type="fabric_idx"/>
+    <item fieldId="1" name="RootPublicKey" type="OCTET_STRING" length="65"/>
+    <item fieldId="2" name="VendorId" type="INT16U"/> <!-- Change INT16U to new type VendorID #2395 -->
+    <item fieldId="3" name="FabricId" type="FABRIC_ID"/>
+    <item fieldId="4" name="NodeId" type="NODE_ID"/>
+    <item fieldId="5" name="Label" type="CHAR_STRING" length="32"/>
   </struct>
 
   <enum name="NodeOperationalCertStatus" type="ENUM8">
     <cluster code="0x003E"/>
-    <item name="SUCCESS" value="0x00"/>
-    <item name="InvalidPublicKey" value="0x01"/>
-    <item name="InvalidNodeOpId" value="0x02"/>
-    <item name="InvalidNOC" value="0x03"/>
-    <item name="MissingCsr" value="0x04"/>
-    <item name="TableFull" value="0x05"/>
-    <item name="InsufficientPrivilege" value="0x08"/>
-    <item name="FabricConflict" value="0x09"/>
-    <item name="LabelConflict" value="0x0a"/>
-    <item name="InvalidFabricIndex" value="0x0b"/>
+    <item fieldId="0" name="SUCCESS" value="0x00"/>
+    <item fieldId="1" name="InvalidPublicKey" value="0x01"/>
+    <item fieldId="2" name="InvalidNodeOpId" value="0x02"/>
+    <item fieldId="3" name="InvalidNOC" value="0x03"/>
+    <item fieldId="4" name="MissingCsr" value="0x04"/>
+    <item fieldId="5" name="TableFull" value="0x05"/>
+    <item fieldId="6" name="MissingACL" value="0x06"/>
+    <item fieldId="7" name="MissingIpk" value="0x07"/>        
+    <item fieldId="8" name="InsufficientPrivilege" value="0x08"/>
+    <item fieldId="9" name="FabricConflict" value="0x09"/>
+    <item fieldId="10" name="LabelConflict" value="0x0a"/>
+    <item fieldId="11" name="InvalidFabricIndex" value="0x0b"/>
   </enum>
 
   <struct name="NOCStruct">
     <cluster code="0x003E"/>
-    <item name="FabricIndex" type="INT8U"/>
-    <item name="NOC" type="OCTET_STRING"/>
+    <item fieldId="0" name="FabricIndex" type="fabric_idx"/>
+    <item fieldId="1" name="NOC" type="OCTET_STRING" length="400" isFabricSensitive="true"/>
```

Comment:
Is `isFabricSensitive` an alias for the equivalent of an `access` tag?

Reply:
It has been suggested to use on Nov 1st: https://csamembers.slack.com/archives/C02FTN42KEE/p1635797498010800?thread_ts=1635795804.009300&cid=C02FTN42KEE.

Reply:
This was suggested on Nov 1st: https://csamembers.slack.com/archives/C02FTN42KEE/p1635797498010800?thread_ts=1635795804.009300&cid=C02FTN42KEE